src/modules/blockchainApplication/connection/components/ConnectionSummary/index.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import { withRouter } from 'react-router';
+import ValueAndLabel from 'src/modules/transaction/components/TransactionDetails/valueAndLabel';
+import { PrimaryButton, SecondaryButton } from 'src/theme/buttons';
+import { EVENTS } from '@libs/wcm/constants/lifeCycle';
+import { addSearchParamsToUrl } from 'src/utils/searchParams';
+import ConnectionContext from '@libs/wcm/context/connectionContext';
+import useSession from '@libs/wcm/hooks/useSession';
+import { useAccounts } from '@account/hooks';
+import BlockchainAppDetailsWrapper from '../../../explore/components/BlockchainAppDetailsWrapper';
+import AccountsSelector from './AccountsSelector';
+import styles from './connectionSummary.css';
+
+// eslint-disable-next-line max-statements
+const ConnectionSummary = ({ history }) => {
+  const [addresses, setAddresses] = useState([]);
+  const { t } = useTranslation();
+  const { accounts } = useAccounts();
+  const { events } = useContext(ConnectionContext);
+  const { approve, reject } = useSession();
+
+  const connectHandler = async () => {
+    const status = await approve(addresses);
+    addSearchParamsToUrl(history, { modal: 'connectionSuccess', status });
+  };
+
+  const rejectHandler = () => {
+    reject();
+  };
+
+  // istanbul ignore next
+  if (!events.length || events[events.length - 1].name !== EVENTS.SESSION_PROPOSAL) {
+    return <div>{t('Connection summary is not ready yet.')}</div>;
+  }
+
+  const { proposer, requiredNamespaces, pairingTopic } = events[events.length - 1].meta.params;
+  const application = {
+    data: {
+      name: proposer.metadata.name,
+      serviceUrl: proposer.metadata.url.replace(/\/$/, ''),
+      icon: proposer.metadata.icons[0],
+      address: `Chain ID: ${requiredNamespaces.lisk.chains[0].replace('lisk:', '')}`,
+    },
+  };
+
+  return (
+    <BlockchainAppDetailsWrapper
+      application={application}
```

Comment:
if you want to pass extra information you dont need to change entire BlockchainAppDetails to an other component then pass somethings as children (its not reusable)
pass extra part as props to the original component

Reply:
Done.